modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/node/CheckNodeExistsCommand.java

```diff
@@ -61,7 +64,7 @@ else if ( !found.equals( skipThrowIfSameId ) )
 
         private boolean throwIfExists;
 
-        private NodeId skipThrowIfSameId;
+        private boolean refresh = false;
```

Comment:
Maybe should be true by default, and only set to false where we prefer performance over consistency (load/import)

Maybe give it a different name and join with use of primary preference. Need to discuss

Reply:
MODE Accuracy vs Speed